microservices/fe/thread_request.py

```diff
@@ -7,7 +7,7 @@
 from queue import Queue
 from datetime import datetime
 from pathlib import Path
-from config import W, H, FPS, urlapi, CORAL_DATA_DIR, img_format
+from fe.config import W, H, FPS, urlapi, CORAL_DATA_DIR, img_format
 
 class Request_api (threading.Thread):
```

Comment:
Naming convention for classes usually doesn't have underscore (similarly to what you guys were doing elsewhere already): https://www.python.org/dev/peps/pep-0008/#class-names

`RequestAPI` seems to be a good option :)

Reply:
done